RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronProducer.cc

```diff
@@ -0,0 +1,46 @@
+#include "RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronProducer.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectronFwd.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectronCore.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectronCoreFwd.h"
+#include "DataFormats/GsfTrackReco/interface/GsfTrack.h"
+#include "DataFormats/GsfTrackReco/interface/GsfTrackFwd.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "RecoEgamma/EgammaElectronAlgos/interface/GsfElectronAlgo.h"
+#include <iostream>
+
+using namespace reco;
+
+LowPtGsfElectronProducer::LowPtGsfElectronProducer( const edm::ParameterSet& cfg, 
+						    const gsfAlgoHelpers::HeavyObjectCache* hoc )
+  : GsfElectronBaseProducer(cfg,hoc)
+{}
+
+LowPtGsfElectronProducer::~LowPtGsfElectronProducer()
+{}
+
+void LowPtGsfElectronProducer::produce( edm::Event& event, const edm::EventSetup& setup )
+{
+  auto electrons = std::make_unique<GsfElectronCollection>();
+  edm::Handle<reco::GsfElectronCoreCollection> coreElectrons;
+  event.getByToken(inputCfg_.gsfElectronCores,coreElectrons);
+  for ( unsigned int ii=0; ii < coreElectrons->size(); ++ii ) {
+    const GsfElectronCoreRef ref = edm::Ref<GsfElectronCoreCollection>(coreElectrons,ii);
+    GsfElectron* ele = new GsfElectron(ref);
```

Comment:
another memory leak. Dont make this a pointer, just make it a regular object

Reply:
As above, added delete statement [here](https://github.com/CMSBParking/cmssw/pull/3/commits/ea54c5c0d7ea310dd1210818d4176de611977206#diff-ae578e3a881637a06b1fd6804d5c5240R51)